adhocteam/Head-Start-TTADP
# [TTAHUB-101] change app name to TTA Hub

## Description of change

Changed app name on the frontend and in email templates

Not changed (but can be)
- CSS classes (.ttasmarthub, etc)
- Documentation

## How to test

Confirm that both the login modal and the upper left text in the header have the correct text.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-110

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
